pkg/arvo/gen/dojo/grant.hoon

```diff
@@ -0,0 +1,9 @@
+::  grant: allow a ship to link to dojo
+::
+/?    310
+::
+:-  %say
+|=  $:  [now=@da eny=@uvJ bec=beak]
+        [[=ship ~] ~]
+    ==
+[%grant-access ship]
```

Comment:
do we want to print anything here? Maybe even prompting the user to confirm? I'm slightly concerned about unsuspecting users getting tricking into running `:dojo|grant ~scammer`

Reply:
maybe just more explicit `:dojo|allow-remote-login`

Reply:
or `:dojo|trust`

Reply:
(for bonus points, allow e.g. `:dojo|trust 0c1A1zP1eP5QGefi2DMPTfTL5SLmv7DivfNa` for setting up piñata planets, which you redeem as `:~sipnym-sipnym/dojo|redeem-trust our .^(@ cx+%/private/key)` from a comet, sending a message signed with the BTC address key that the sending comet be allowed to log in)

Reply:
I like the `:dojo|trust` or `:dojo|allow-remote-login`

Reply:
Let's do `allow-remote-login`. This isn't an operation that should be done frequently or lightly, so a long, scary name is preferable to a short one with multiple interpretations.